### PR TITLE
[WIP] Prototype before you ship checklist

### PR DESCRIPTION
**Pull request by alain-hoang**
This changeset adds a template generator for generating a base ATO checklist adapted from the  [Before You Ship guide](https://pages.18f.gov/before-you-ship/).

### CODE REVIEW

File: ato/index.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	content = content.replace('    * <url>\n    * ...',
+			'    * ' + this.config.get('runningLibs'));
+	content = content.replace('Site:** <url>',
+			'Site:** ' + this.config.get('siteName'));
+	content = content.replace('Product manager:** @<username>',
+			'Product manager:** ' + this.config.get('productManager'));
+	content = content.replace('Technical point of contact:** @<username>',
+			'Technical point of contact:** ' + this.config.get('technicalPOC'));
+	content = content.replace('Launch date/deadline:** <date>',
+			'Launch date/deadline:** ' + this.config.get('launchDate'));
```
Comment (cmc333333) [inline ato/index.js:94]: The slice and replacements here seem fragile. Would it make more sense to make the original document easier to parse and use that here? For example:

```md
### Checklist
* **Main repository:** <mainRepositoryUrl>
* ...
```

File: ato/index.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      });
+    }
+
+    if (!this.config.get('runningLibs')) {
+      prompts.push({
+        type: 'input',
+        name: 'runningLibs',
+        message: 'List the known running libraries for this site.'
+      });
+    }
```
Comment (cmc333333) [inline ato/index.js:22]: Should this be a comma separated list or can it ask for more than one library?

**Review by cmc333333 — COMMENTED** [review]
Nice @alain-hoang. I think this is a valuable direction, but want to check with the ATO folks to make sure they'd actually use this (as opposed to .about.yml or some other format).

**Comment by cmc333333** [comment]
@alain-hoang after chatting with @afeld about this, it looks like there's some confusion. This checklist is for _creating issues_; it shouldn't be in the project's code base. That said, a very similar file (.about.yml) _should_ be in the project (it's actually one of the steps in the todo list). Maybe we can generate that instead?

File: ato/index.js (hunk 2 above)
Comment (alain-hoang) [reply ato/index.js:22]: The intention for this was to allow more than one library.

**Comment by alain-hoang** [comment]
Thank you for the comments and feedback @cmc333333 especially on the clarifications regarding the .about.yml!  I plan to look more into the .about.yml file format.   Closing out this pull request, for now.

**Pull request closed without merging**